typescript.js

```diff
@@ -1,12 +1,13 @@
 module.exports = {
   extends: [
-    'eslint:recommended',
+    "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "prettier",
     "plugin:prettier/recommended",
     "prettier/@typescript-eslint",
   ],
   parser: "@typescript-eslint/parser",
+  parserOptions: { project: "./tsconfig.json" },
```

Comment:
This configuration should be the responsibility of the project using the config, IMO.